src/build_queue.rs

```diff
@@ -652,12 +654,191 @@ impl BuildQueue {
     }
 }
 
+/// Queue rebuilds as configured.
+///
+/// The idea is to rebuild:
+/// * the latest release of each crate
+/// * when the nightly version is older than our configured threshold
+/// * and there was a successful build for that release, that included documentation.
+/// * starting with the oldest nightly versions.
+/// * also checking if there is already a build queued.
+///
+/// This might exclude releases from rebuilds that
+/// * previously failed but would succeed with a newer nightly version
+/// * previously failed but would succeed just with a retry.
+#[instrument(skip_all)]
+pub async fn queue_rebuilds(
+    conn: &mut sqlx::PgConnection,
+    config: &Config,
+    build_queue: &AsyncBuildQueue,
+) -> Result<()> {
+    let already_queued_rebuilds = sqlx::query_scalar!(
+        r#"SELECT COUNT(*) as "count!" FROM queue WHERE priority >= $1"#,
+        REBUILD_PRIORITY
+    )
+    .fetch_one(&mut *conn)
+    .await?;
+
+    let rebuilds_to_queue = config
+        .max_queued_rebuilds
+        .expect("config.max_queued_rebuilds not set") as i64
```

Comment:
You're sure about this `expect`? Don't want to log an error and return early instead?

Reply:
yes, the whole method is not even called when the config var is missing.